simulation/tests/course_world_model/test_demo_script.py

```
@@ -0,0 +1,228 @@
+    ):
+        first_dir, _ = first_run
+        unexpected = [
+            str(path.relative_to(first_dir))
+            for path in first_dir.iterdir()
+            if path.name != "pilot-course-a"
+        ]
+        assert unexpected == []
+
+    def test_no_generated_artifacts_appear_in_the_repository(self, first_run):
+        result = subprocess.run(
+            ["git", "status", "--porcelain", "--", "reports"],
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)